retrofit/src/main/java/retrofit2/Response.java

```diff
@@ -85,6 +85,7 @@
   public static <T> Response<T> error(int code, ResponseBody body) {
     if (code < 400) throw new IllegalArgumentException("code < 400: " + code);
     return error(body, new okhttp3.Response.Builder() //
+        .body(body)
```

Comment:
This needs to be changed to mimic:

https://github.com/square/retrofit/blob/master/retrofit/src/main/java/retrofit2/OkHttpCall.java#L204